src/Compatibility/Microsoft.DotNet.ApiCompatibility/Rules/EnumValuesMustMatch.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Microsoft.CodeAnalysis;
+using Microsoft.DotNet.ApiCompatibility.Abstractions;
+
+namespace Microsoft.DotNet.ApiCompatibility.Rules
+{
+    public class EnumValuesMustMatch : Rule
+    {
+        public override void Initialize(RuleRunnerContext context)
+        {
+            context.RegisterOnTypeSymbolAction(RunOnTypeSymbol);
+        }
+
+        private static bool isEnum(ITypeSymbol type)
+        {
+            return type != null && type.IsType && type.TypeKind == TypeKind.Enum;
+        }
+        private void RunOnTypeSymbol(ITypeSymbol left, ITypeSymbol right, string leftName, string rightName, IList<CompatDifference> differences)
```

Comment:
```suggestion
        private static bool isEnum(ITypeSymbol type)
            => type != null && 
                  type.IsType && 
                  type.TypeKind == TypeKind.Enum;

        private void RunOnTypeSymbol(ITypeSymbol left, ITypeSymbol right, string leftName, string rightName, IList<CompatDifference> differences)
```
Same here. Do we actually have to check for `type.IsType`? Is there ever a scenario where ITypeSymbol won't be a type?

Reply:
I think you're right. I might have gotten confused because I saw that it inherits `INamespaceOrTypeSymbol`.